src/Styles/Header.scss

```diff
@@ -0,0 +1,112 @@
+ul {
+  navelement {
+    font-weight: 700;
+    font-family: Open Sans,sans-serif;
+  }
+}
+.header {
+  position: relative;
+  width: 100%;
+  background-color: white;
+}
+.header {
+  .navigation {
+    padding: 8px 0;
+    position: relative;
+    border-bottom: 1px solid #ebf2f2;
+  }
+}
+.container {
+  padding: 0 20px;
+  max-width: 1140px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.container > ul {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  list-style-type: none;
+  width: 705px;
+  padding: 0;
+}
+.container {
+  div {
+    img {
+      height: 48px;
+      max-width: 250px;
+    }
+  }
+}
+.navElementContainer {
+  position: relative;
+  padding-top: 15px;
+  cursor: pointer;
+}
+.navElementContainer {
+  li {
+   div {
+     span {
+       transition-duration: 0.3s;
+       font-weight: 700;
+     }
+   }
+  }
+}
```

Comment:
What in the world is that abomination? Why not just apply style directly to the element?